### DISCUSSION TITLE
Disable showing starting message when toggling chat window

### DISCUSSION THREAD

**Discussion by TroySigX** [logancyang/obsidian-copilot#1171] [Q&A]
**Is your feature request related to a problem? Please describe.**
Whenever I toggle the chat window, this message appears:
![image](https://github.com/user-attachments/assets/3302c054-83ad-4166-9bcf-6ada8b73b289)

This interrupts my workflow a lot since I usually close the chat window to look at my notes and every time I open the window it kinda gets in the way.

**Describe the solution you'd like**
The message should only appear the first time I toggle, after that, it shouldn't be triggered on toggle anymore.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by logancyang** [comment]
What's this message, is it a response to your question? The only message that's hardcoded is the one when you switch to Vault QA mode. Not sure I understand your issue.

**Comment by TroySigX** [comment]
Oh sorry, my bad. This is the message that appears every time:
![image](https://github.com/user-attachments/assets/51a3e8ef-37e1-4a74-8831-435e24860f6b)

**Comment by TroySigX** [comment]
My default mode is vault QA, and every time I toggle the chat window, this message appears

**Comment by logancyang** [comment]
Yep we are getting rid of this message in the next iteration!

**Comment by TroySigX** [comment]
Okay thank you!
